challenges/2021_cloud_base_height/03_pytorch_train.ipynb

```diff
@@ -0,0 +1,1039 @@
+{
```

Comment:
<p>the reason for the existence of a data loader might be worth giving some more explanation, I imagine many people will not have com across this before.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MetOffice/data_science_cop/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>